pravega/src/main/scala/akka/stream/alpakka/pravega/settings.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.pravega
+import io.pravega.client.stream.Serializer
+
+final class ReaderSettings[A](val groupName: String,
+                              val readerId: String,
+                              val serializer: Serializer[A],
+                              val timeout: Long = 5000) {}
```

Comment:
This can be represented as a Java duration.  In config the default could be `timeout = 5 seconds`

Reply:
bump